app/repository/rest/RestRepository.scala

```diff
@@ -93,14 +93,14 @@ class RestRepository @Inject() (config: RestRepositoryConfig, wsClient: TraceWSC
     }
   }
 
-  private def withTranslationOfFailureToEditStatus = new PartialFunction[Throwable, EditParentLinkStatus] {
+  private def withTranslationOfFailurePatchStatus = new PartialFunction[Throwable, PatchStatus] {
     override def isDefinedAt(cause: Throwable): Boolean = true
 
-    override def apply(cause: Throwable): EditParentLinkStatus = {
+    override def apply(cause: Throwable): PatchStatus = {
       logger.error(s"Translating unit request failure [$cause].")
       cause match {
-        case t: TimeoutException => EditFailure
-        case t: Throwable => EditFailure
+        case t: TimeoutException => PatchFailure
+        case t: Throwable => PatchFailure
```

Comment:
Your IDE and warnings will probably be telling you that ```t``` is unused.  You can use ```_``` here.